e2e_test/source/basic/kafka.slt

```diff
@@ -15,9 +15,20 @@ create materialized source s1 (v1 int, v2 varchar) with (
   scan.startup.mode = 'earliest'
 ) row format json
 
+statement ok
+create source ps1 (v1 int, v2 varchar) with (
+  connector = 'kafka',
+  topic = 'kafka_1_partition_topic',
+  properties.bootstrap.server = '127.0.0.1:29092',
+  scan.startup.mode = 'earliest'
+) row format json
+
 statement ok
 select * from s1;
 
+statement ok
+select * from ps1;
```

Comment:
We do it in streaming scenes to ensure the job is started. We don't check the answer here because the source exec may not finish loading all records.

I don't think we need it for batch queries